packages/react-core/src/components/Form/examples/FormFieldGroups.tsx

```diff
@@ -0,0 +1,388 @@
+import React from 'react';
+import {
+  Form,
+  FormGroup,
+  FormFieldGroup,
+  FormFieldGroupExpandable,
+  FormFieldGroupHeader,
+  TextInput,
+  Button
+} from '@patternfly/react-core';
+import TrashIcon from '@patternfly/react-icons/dist/esm/icons/trash-icon';
+
+export const FieldGroups: React.FunctionComponent = () => {
+  const initialValues = {
+    'form-expandable-field-groups-label1': '',
```

Comment:
Is there any way we could clean these names up? Would be a lot more DRY and readable if we removed the `form-expandable-field-groups-` text from all of these!
```suggestion
    'label1': '',
```

Reply:
Looks good and I totally agree @jenny-s51 ! The only issue I ran into with this was that the names of the labels in `initialValues` had to match the `id` and other props in the components used in order for the state variables to update the input. If I'm able to, I can change all of the id props to match it to something simpler?

Reply:
That sounds like a good plan @andyyvo !